docs/Researcher/scheduling/using-node-pools.md

```diff
@@ -13,7 +13,7 @@ The *Node Pools* feature is enabled by default:
 
 Once the feature is enabled by the administrator, all nodes in each of your upgraded clusters are associated with the `Default` node pool.
 
-## Creating and Using Node-Pools
+## Creating and using Node-Pools
```

Comment:
Are you sure you want to use the word "using" and not "taking advantage of"

Reply:
Yes, I'm sure